LayoutTests/fast/css/content-visibility-zoom.html

```diff
@@ -0,0 +1,70 @@
+<!doctype html>
+<html>
+<head>
+    <title>content-visibility demo</title>
+    <style>
+        body {
+            zoom: 60%;
```

Comment:
Should we also have a test that uses `window.internals.setPageZoomFactor(2);`?

Reply:
I think this test is enough.